library/clojure

```diff
@@ -2,72 +2,57 @@ Maintainers: Paul Lam <paul@quantisan.com> (@Quantisan),
              Wes Morgan <wesmorgan@icloud.com> (@cap10morgan)
 Architectures: amd64
 GitRepo: https://github.com/Quantisan/docker-clojure.git
-GitCommit: e96f6249cac065855a40a9f36ba88949e0a52d7d
+GitCommit: 09a243f78bde5ce2f5304b4d10a5d6b8152dc803
 
-Tags: openjdk-8, openjdk-8-lein, openjdk-8-lein-2.9.1, openjdk-8-stretch, openjdk-8-lein-stretch, openjdk-8-lein-2.9.1-stretch
-Directory: target/openjdk-8-stretch/lein
+Tags: latest
+Directory: target/openjdk-11-slim-buster/latest
 
-Tags: openjdk-8-slim-buster, openjdk-8-lein-slim-buster, openjdk-8-lein-2.9.1-slim-buster
+Tags: openjdk-8, openjdk-8-lein, openjdk-8-lein-2.9.2, openjdk-8-buster, openjdk-8-lein-buster, openjdk-8-lein-2.9.2-buster
+Directory: target/openjdk-8-buster/lein
+
+Tags: openjdk-8-slim-buster, openjdk-8-lein-slim-buster, openjdk-8-lein-2.9.2-slim-buster
 Directory: target/openjdk-8-slim-buster/lein
 
-Tags: openjdk-8-boot, openjdk-8-boot-2.8.3, openjdk-8-boot-stretch, openjdk-8-boot-2.8.3-stretch
-Directory: target/openjdk-8-stretch/boot
+Tags: openjdk-8-boot, openjdk-8-boot-2.8.3, openjdk-8-boot-buster, openjdk-8-boot-2.8.3-buster
+Directory: target/openjdk-8-buster/boot
 
 Tags: openjdk-8-boot-slim-buster, openjdk-8-boot-2.8.3-slim-buster
 Directory: target/openjdk-8-slim-buster/boot
 
-Tags: openjdk-8-tools-deps, openjdk-8-tools-deps-1.10.1.502, openjdk-8-tools-deps-stretch, openjdk-8-tools-deps-1.10.1.502-stretch
-Directory: target/openjdk-8-stretch/tools-deps
+Tags: openjdk-8-tools-deps, openjdk-8-tools-deps-1.10.1.536, openjdk-8-tools-deps-buster, openjdk-8-tools-deps-1.10.1.536-buster
+Directory: target/openjdk-8-buster/tools-deps
 
-Tags: openjdk-8-tools-deps-slim-buster, openjdk-8-tools-deps-1.10.1.502-slim-buster
+Tags: openjdk-8-tools-deps-slim-buster, openjdk-8-tools-deps-1.10.1.536-slim-buster
 Directory: target/openjdk-8-slim-buster/tools-deps
 
-Tags: openjdk-11, openjdk-11-lein, openjdk-11-lein-2.9.1, lein, lein-2.9.1, openjdk-11-stretch, openjdk-11-lein-stretch, openjdk-11-lein-2.9.1-stretch, lein-stretch, lein-2.9.1-stretch
+Tags: openjdk-11, openjdk-11-lein, openjdk-11-lein-2.9.2, lein, lein-2.9.2, openjdk-11-buster, openjdk-11-lein-buster, openjdk-11-lein-2.9.2-buster, lein-buster, lein-2.9.2-buster
```

Comment:
Some, like this one, seem to contain lein 2.9.3, but is labelled 2.9.2 here :wink:

Reply:
Oh geez, sorry. This is what happens when I let things linger for this long. :) Fixed!